packages/material-ui/src/SvgIcon/SvgIcon.test.js

```diff
@@ -39,55 +38,70 @@ describe('<SvgIcon />', () => {
   );
 
   it('renders children by default', () => {
-    const wrapper = shallow(<SvgIcon>{path}</SvgIcon>);
-    expect(wrapper.contains(path)).to.equal(true);
-    expect(wrapper.props()['aria-hidden']).to.equal(true);
+    const { container, getByTestId } = render(<SvgIcon>{path}</SvgIcon>);
+    const root = container.querySelector(`.${classes.root}`);
+
+    expect(getByTestId('test-path')).to.not.equal(null);
```

Comment:
`getBy*` queries will never return null. They error if no elements match.

Reply:
@joshwooding thanks. I'll replace it with `queryBy*`